src/worker/core/team/valueChange.ts

```diff
@@ -588,4 +624,53 @@ const valueChange = async (
 	return valuesAdd - valuesRemove;
 };
 
+const getModifiedPickRank = async (
+	tid: number,
+	pidsAdd: number[],
+	pidsRemove: number[],
+) => {
+	const teamSeason = await idb.cache.teamSeasons.indexGet(
+		"teamSeasonsBySeasonTid",
+		[tid, g.get("season")],
+	);
+	const teams = g.get("numActiveTeams");
+	const gp = teamSeason?.gp ?? 0;
+	const seasonFraction = gp / g.get("numGames");
+
+	const players = await idb.cache.players.indexGetAll("playersByTid", tid);
+	const playersAfterTrade = players.filter(p => !pidsRemove.includes(p.pid));
+	for (const pid of pidsAdd) {
+		const p = await idb.cache.players.get(pid);
+		if (p) {
+			playersAfterTrade.push(p);
+		}
+	}
+	const playerRatings = playersAfterTrade.map(p => ({
+		pid: p.pid,
+		value: p.value,
+		ratings: {
+			ovr: p.ratings.at(-1)!.ovr,
+			ovrs: p.ratings.at(-1)!.ovrs,
+			pos: p.ratings.at(-1)!.pos,
+		},
+	}));
+
+	const newTeamOvr = team.ovr(playerRatings);
+	const newTeamOvrRank =
+		newTeamOvr < cache.teamOvrs[cache.teamOvrs.length - 1].ovr
+			? cache.teamOvrs.length - 1
```

Comment:
Since we are not removing this team from the array when doing `findIndex` on the next line, this means that we'll have the same value of `newTeamOvrRank` (numActiveTeams - 1) for for when the trade makes the team have either the worst or 2nd worst ovr in the league. I think.

Reply:
If that's correct, could change this to just `cache.teamOvrs.length` and get rid of the -1 on line 663 I think. Also should have some comment explaining the range of values that `newTeamOvrIndex` has cause it's confusing!

Or, maybe a bit slower but less confusing, remove the current team from the list of team ovrs before computing `newTeamOvrIndex`.